src/main/java/mat/client/shared/MeasureLibraryResultTable.java

```diff
@@ -238,12 +238,12 @@ private SafeHtml getMeasureNameColumnToolTip(ManageMeasureSearchModel.Result obj
             sb.appendHtmlConstant("<div class=\"pull-left\">")
                     .appendHtmlConstant(editState)
                     .appendHtmlConstant("<button id='div1' type=\"button\" title=\""
-                            + object.getName() + "\" tabindex=\"-1\" class=\" " + cssClass + "\"></button>")
+                            + SafeHtmlUtils.htmlEscape(object.getName()) + "\" tabindex=\"-1\" class=\" " + cssClass + "\"></button>")
                     .appendHtmlConstant("</div>");
 
         }
         sb.appendHtmlConstant("<div class=\"pull-left\" title=\" Click to open "
-                + object.getName() + "\" tabindex=\"0\">" + object.getName() + "</div>");
+                + SafeHtmlUtils.htmlEscape(object.getName()) + "\" tabindex=\"0\">" + SafeHtmlUtils.htmlEscape(object.getName()) + "</div>");
         return sb.toSafeHtml();
```

Comment:
btw. it already has sb.toSafeHtml. Does it do the same thing as htmlEscape?

Reply:
appendHtmlConstant will take in HTML content without escaping special chars.

Reply:
but now about "sb.toSafeHtml()"

Reply:
toSafeHtml doesn't escape special chars, it Returns the safe HTML accumulated in the builder 'sb'. sb.appendEscaped Appends a string after HTML-escaping it.

Reply:
good to know. thank you